packages/validator/src/services/attestation.ts

```diff
@@ -258,7 +258,7 @@ export class AttestationService {
     }
 
     this.logger.verbose("Aggregating attestations", logCtx);
-    const res = await this.api.validator.getAggregatedAttestation({
+    const res = await this.api.validator.getAggregatedAttestationV2({
```

Comment:
same to the above, `vc` should follow the spec to call correct api depending on the clock slot. Note that lodestar `vc` could call `bn` from other clients as well

Reply:
> same to the above, `vc` should follow the spec to call correct api depending on the clock slot. Note that lodestar `vc` could call `bn` from other clients as well

@twoeths Since this branch is not production ready, we shouldn't use have any other vc calling our bn. For the purpose of devnet-1, I need to communicate with ethpandaops to not pull the latest commit but a specific commit.